readme.md

```diff
@@ -27,6 +27,9 @@ camelcaseKeys({'foo-bar': true, nested: {unicorn_rainbow: true}}, {deep: true});
 camelcaseKeys({a_b: 1, a_c: {c_d: 1, c_e: {e_f: 1}}}, {deep: true, stopPaths: ['a_c.c_e']}),
 //=> {aB: 1, aC: {cD: 1, cE: {e_f: 1}}}
 
+camelcaseKeys({'foo-BAR': true, nested: {unicorn_RAINbow: true}}, {deep: true, preserveConsecutiveUppercase: true});
+//=> {fooBAR: true, nested: {unicornRAINbow: true}}
```

Comment:
This should be on line 125, together with the docs for it.

Reply:
I kept the way it was made for the other options. At the moment it is not the case for all the option. It is also consistent with your other related packages (i.e. camelcase README). 
=> Is it ok to keep it like this at the moment? I've added an example with the value = false as well.